frame/bridge/eth/relay/src/lib.rs

```diff
@@ -123,31 +119,44 @@ decl_storage! {
 		/// Hash of best block header
 		pub BestHeaderHash get(fn best_header_hash): H256;
 
-		pub CanonicalHeaderHashOf get(fn canonical_header_hash_of): map hasher(identity) u64 => H256;
+		pub CanonicalHeaderHashes get(fn canonical_header_hash): map hasher(identity) u64 => H256;
 
-		pub HeaderOf get(fn header_of): map hasher(identity) H256 => Option<EthHeader>;
-
-		pub HeaderInfoOf get(fn header_info_of): map hasher(identity) H256 => Option<HeaderInfo>;
+		pub Headers get(fn header): map hasher(identity) H256 => Option<EthHeader>;
+		pub HeaderBriefs get(fn header_brief): map hasher(identity) H256 => Option<EthHeaderBrief>;
 
 		/// Number of blocks finality
 		pub NumberOfBlocksFinality get(fn number_of_blocks_finality) config(): u64;
 		pub NumberOfBlocksSafe get(fn number_of_blocks_safe) config(): u64;
 
-		pub CheckAuthorities get(fn check_authorities) config(): bool = true;
+		pub CheckAuthority get(fn check_authority) config(): bool = true;
 		pub Authorities get(fn authorities) config(): Vec<T::AccountId>;
 	}
 	add_extra_genesis {
 		// genesis: Option<Header, Difficulty>
 		config(genesis_header): Option<(u64, Vec<u8>)>;
-		config(dag_merkle_roots): Vec<H128>;
+		config(dag_merkle_roots): DagMerkleRoots;
```

Comment:
DagMerkleRoots is defined in std, can be used here?

Reply:
https://substrate.dev/docs/en/1.0/runtime/types/genesisconfig-struct#setting-the-genesis-configuration-values